api/pkg/apis/v1alpha1/vendors/campaigns-vendor_test.go

```diff
@@ -75,7 +75,7 @@ func TestCampaignsOnCampaigns(t *testing.T) {
 		Method: fasthttp.MethodPost,
 		Body:   data,
 		Parameters: map[string]string{
-			"__name": "campaign1",
+			"__name": "campaign1-v1",
```

Comment:
I am guessing this should be ':', since this is in path parameter.

Reply:
Create operation. Why :?

Reply:
Can you please list the routes we changed on all vendors?

Reply:
We do not change routes .